api/specs/common/schemas/node-meta-v0.0.1.json

```diff
@@ -248,6 +248,11 @@
                 true
               ]
             },
+            "unit": {
+              "title": "Unit",
+              "description": "Units of this input value, if a physical quantity",
+              "type": "string"
```

Comment:
one question here. are we accepting whatever units or only SI base units? I mean is there any control planned?
Also, how do you define no units? with an empty string or an empty payload?

Reply:
yes, the controlled is going to be done in the backend (still not implemented) using pint library. It has a very powerful string parser for units and together with the metadata, we plan to build a parser and validator of units in coming PR

yes, no units is defined with an empty string or simply not setting it. Actually, the test ``test_services.py::test_service_port_units`` receives an empty string and pint resolves it as ``dimensionless``